channel_test.go

```diff
@@ -411,3 +422,29 @@ func TestChannel_RejectInvite(t *testing.T) {
 	err = ch.RejectInvite(members[0], &Message{Text: "rejected", User: &User{ID: members[0]}})
 	require.NoError(t, err, "reject invite")
 }
+
+func ExampleChannel_Update() {
+	// https://getstream.io/chat/docs/channel_permissions/?language=python
+	client := &Client{}
+
+	data := map[string]interface{}{
+		"image":      "https://path/to/image",
+		"created_by": "elon",
+		"roles":      map[string]string{"elon": "admin", "gwynne": "moderator"},
```

Comment:
roles can be a bit confusing (since you are adding a channel and role is a user property )

Reply:
I'm not sure I understand what you are saying. The example was transliterated from the python. So this is just what the python one does 😁